src/main/starlark/builtins_bzl/common/python/py_runtime_rule.bzl

```diff
@@ -77,6 +77,7 @@ def _py_runtime_impl(ctx):
             coverage_files = coverage_files,
             python_version = python_version,
             stub_shebang = ctx.attr.stub_shebang,
+            bootstrap_template = ctx.attr.bootstrap_template,
```

Comment:
This should be ctx.file.bootstrap_template. Did tests catch this? The starlark py_runtime rule implementation should be enabled by default...is that not what you're seeing?

Reply:
All tests are passing, but I also haven't added any tests for the starlark codepath yet. I was going to ask which path I should be adding the most tests for, and I think you replied recently saying the Java path is going away soon so I'll add the tests to starlark

Reply:
The tests implementated in java should be using this starlar -- oh I know what it is. The py_runtime _rule_ is enabled by default, but the PyRuntimeInfo _provider_ isn't enabled by default yet (I was in the middle of that when I noticed py_binary still had a reference to it).

So go ahead and change this to ctx.file since we know that's the correct thing to do, but this starlark-ctor code here isn't yet used; no need to block this PR on writing tests for it.